pkg/api/node/formatter.go

```diff
@@ -290,7 +290,29 @@ func (h ActionHandler) powerActionPossible(rw http.ResponseWriter, node string)
 		}
 		return err
 	}
-	rw.WriteHeader(http.StatusOK)
+
+	val, ok, err := unstructured.NestedString(inventoryObject.Object, "status", "status")
+	if err != nil {
+		logrus.Errorf("error fetching status from inventory object %s :%v", node, err)
+		rw.WriteHeader(http.StatusFailedDependency)
```

Comment:
Prefer to use http code 409 instead.
Refer to https://http.dev/424 and https://http.dev/409